CMakeLists.txt

```diff
@@ -311,7 +314,7 @@ endif()
 # FIXME: Support ARM "-mfpu=vfp -mfloat-abi=hard"
 if (("${TARGET_ARCHITECTURE}" STREQUAL "x86_64") OR ("${TARGET_ARCHITECTURE}" STREQUAL "i686"))
   if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
-    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2")
+    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2" "-std=c++0x")
```

Comment:
@NikolajBjorner The `-std=c++0x` flag doesn't really belong here. It isn't related to SSE flags.

Is Z3 moving to C++11 now?